x/leverage/keeper/keeper_test.go

```diff
@@ -1193,29 +1131,6 @@ func (s *IntegrationTestSuite) TestGetEligibleLiquidationTargets_TwoAddr() {
 	s.Require().Equal([]sdk.AccAddress{lenderAddr, anotherLender}, lenderAddress)
 }
 
-func (s *IntegrationTestSuite) TestExchangeRatesInvariant() {
```

Comment:
Note: Removed test as manually setting exchange rates is no longer possible (they are always derived instantly)